core/src/main/resources/error/error-classes.json

```diff
@@ -703,6 +725,11 @@
     ],
     "sqlState" : "42805"
   },
+  "GROUP_EXPRESSION_TYPE_IS_NOT_ORDERABLE" : {
+    "message" : [
+      "expression <sqlExpr> cannot be used as a grouping expression because its data type <dataType> is not an orderable data type."
```

Comment:
```suggestion
      "The expression <sqlExpr> cannot be used as a grouping expression because its data type <dataType> is not an orderable data type."
```